src/main.rs

```diff
@@ -0,0 +1,171 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::StreamExt;
+use  std::fs;
+use text_io::read;
+use std::io;
+
+#[derive(Debug)]
+enum HandleError {
+    Io(io::Error),
+    TypeDB(typedb_client::error::Error),
+}
+
+const TEST_DATABASE: &str = "codeforces-data-model";
+ 
+fn new_core_connection() -> typedb_client::Result<Connection> {
+    Connection::new_plaintext("localhost:1729")
+}
+
+async fn load_data(connection: Connection)->Result<(), HandleError>{
+    let data=fs::read_to_string("./src/data.tql").map_err(HandleError::Io)?;    
+    let databases = DatabaseManager::new(connection.clone());
+    let session = Session::new(databases.get(TEST_DATABASE).await.map_err(HandleError::TypeDB)?, Data).await.map_err(HandleError::TypeDB)?;
+    let transaction = session.transaction(Write).await.map_err(HandleError::TypeDB)?;
+    let _ = transaction.query().insert(data.as_str());
+    transaction.commit().await.map_err(HandleError::TypeDB)?;
+    println!("\nData Loaded Successfully\n");
+    Ok(())
+}
+
+async fn load_schema(connection: Connection)->Result<(), HandleError>{
+    let schema = fs::read_to_string("./src/schema.tql").map_err(HandleError::Io)?;
+    let databases = DatabaseManager::new(connection.clone());
+    if databases.contains(TEST_DATABASE).await.map_err(HandleError::TypeDB)?==false {
```

Comment:
Can we make this `if`/`else` check more consise, rather than checking for `== false` ?

Reply:
Switching the if else condition so as to avoid this ugly " == false " condition